doc/whats_new/v0.20.rst

```diff
@@ -7,6 +7,50 @@
 Version 0.20.3
 ==============
 
+**July 29, 2019**
+
+This is a bug-fix release with some bug fixes applied to version 0.20.3.
+
+Changelog
+---------
+
+The bundled version of joblib was upgraded from 0.13.0 to 0.13.2.
+
+:mod:`sklearn.cluster`
+..............................
+
+- |Fix| Fixed a bug in :class:`cluster.KMeans` where KMeans++ initialisation
+  could rarely result in an IndexError. :issue:`11756` by `Joel Nothman`_.
```

Comment:
this entry is not included in master (and 0.21.3)?

Reply:
Ahh, true. It's now included in the 0.20.4 section in master at least...

Reply:
Sorry for missing it.